corehq/toggles.py

```diff
@@ -1211,3 +1211,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN],
     always_enabled={'icds-cas'}
 )
+
+DISPLAY_CONDITION_ON_NODESET = StaticToggle(
```

Comment:
Would rename this now that it applies to tabs whether or not they have nodesets.